mon-pix/app/controllers/campaigns/join-restricted-campaign.js

```diff
@@ -52,9 +47,9 @@ export default Controller.extend({
   monthOfBirth: '',
   yearOfBirth: '',
   birthdate: computed('yearOfBirth', 'monthOfBirth', 'dayOfBirth', function() {
-    const monthOfBirth = _pad(this.monthOfBirth, 2);
-    const dayOfBirth = _pad(this.dayOfBirth, 2);
-    return [this.yearOfBirth, monthOfBirth, dayOfBirth].join('-');
+    const monthOfBirth = standardizeNumberInTwoDigitFormat(this.monthOfBirth.trim());
+    const dayOfBirth = standardizeNumberInTwoDigitFormat(this.dayOfBirth.trim());
+    return [this.yearOfBirth.trim(), monthOfBirth, dayOfBirth].join('-');
```

Comment:
Pour simplifier la lecture, je nettoierais `this.yearOfBirth.trim()` via une variable comme le jour et le mois.

Reply:
Tu ferais quoi dans le nettoyage ? Parce que pour le mois et le jour on met des zéro au début si l'utilisateur n'a saisi qu'un chiffre. Mais on ne veut pas faire la même chose pour l'année, puisque aucune année ne commence par un zéro...

Reply:
Seulement sortir le trom du tableau, pour pouvoir avoir les différentes modifications hors de l'array pour le join.

Reply:
ok ! :)